getting-started.md

```diff
@@ -72,13 +70,13 @@ If everything works, you should see your starter app on your device or simulator
 <div id="starter-app-screenshots">
   <div class="box2">
     <figure>
-      <img src="/images/flutter-starter-app-ios.png">
+      <img src="/images/flutter-starter-app-ios.png" alt="iOS screenshot">
```

Comment:
Yes, those were not great alt strings.

I don't think omitting the attribute would be correct. From the spec you linked, that is only correct when the image contents are not known.

Rather, in this case the image is *not* a key part of the content, but rather merely a graphical representation of the surrounding text, so they can be left empty per  §4.8.4.4.6.